data/visualize_popularities.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2020 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Visualize the popularity bias for a given model's Probe 1."""
+
+import json
+import os
+
+from absl import app
+from absl import flags
+from absl import logging
+import numpy as np
+import plotly.graph_objects as go
+import tensorflow.compat.v1 as tf
+from trainer import constants
+
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_integer("steps", 1009900, "Finetuning training steps.")
```

Comment:
The flag name is a bit confusing. We are not doing fine-tuning in this code, we are reading a fine-tuned model and specifying on what checkpoint to run the analysis on. Can you update the flag comments to reflect this better? On that note - is it possible to skip specifying the steps and just read the latest available (e.g. if flag value is -1 or similar)?

Reply:
Yeah that's confusing. I'll just get rid of it and make it run on the most recent checkpoint.